.github/workflows/ubuntu-ci-x86_64-gnu.yaml

```diff
@@ -113,20 +117,22 @@ jobs:
             # This allows us to "spin up" builds that altogether take longer than
             # six hours, and/or fail later in the build process.
 
-            # base-env
-            echo "base-env ..."
-            spack install --fail-fast --source --no-check-signature base-env 2>&1 | tee log.install.gnu-11.4.0-buildcache.base-env
-            spack buildcache create -u /home/ubuntu/spack-stack/build-cache/ base-env
-
-            # jedi-base-env
-            echo "jedi-base-env ..."
-            spack install --fail-fast --source --no-check-signature jedi-base-env 2>&1 | tee log.install.gnu-11.4.0-buildcache.jedi-base-env
-            spack buildcache create -u /home/ubuntu/spack-stack/build-cache/ jedi-base-env
-
-            # jedi-ufs-env
-            echo "jedi-ufs-env ..."
-            spack install --fail-fast --source --no-check-signature jedi-ufs-env 2>&1 | tee log.install.gnu-11.4.0-buildcache.jedi-ufs-env
-            spack buildcache create -u /home/ubuntu/spack-stack/build-cache/ jedi-ufs-env
+            if [[ "${TEMPLATE}" == *"unified-dev"* || "${TEMPLATE}" == *"skylab-dev"* ]]; then
```

Comment:
The new cylc-dev template doesn't depend on base-env, jedi-base-env, jedi-ufs-env